lib/fastlane/plugin/wpmreleasetoolkit/actions/common/update_pull_requests_milestone_action.rb

```diff
@@ -0,0 +1,104 @@
+require 'fastlane/action'
+require_relative '../../helper/github_helper'
+
+module Fastlane
+  module Actions
+    class UpdatePullRequestsMilestoneAction < Action
```

Comment:
PS: I'm open to bike-shedding the name of this action.
 - `update_milestone` didn't seem correct, especially because the GitHub API named "update milestone" is about updating the milestone itself (its title, due date, …), not about updating the milestone associated with PRs.
 - On the other hand, `update_pull_requests_milestone` is a bit of a mouthful (well not the longest action name we have though, so maybe not so bad). Maybe `update_prs_milestone` would be ok too?
 - Especially given that an alternate idea like `update_milestone_of_opened_pull_requests` would be equally long and convoluted naming.
 - I was torn by using plural or singular for `*_pull_requests_*` in the name, given technically this action can also update a single PR. But I figured since the most common use case we'll have will be updating _all_ opened PRs in a given milestone, it made more sense to use plural

Reply:
Dropping one more suggestion: `update_milestone_for_prs`.

But I think the name is fine; its shorter version, `update_prs_milestone`, doesn't sound bad either.